snmp/__init__.py

```diff
@@ -60,7 +65,7 @@ def __init__(self, sh, *args, **kwargs):
         self.instance = self.get_parameter_value('instance')         # the instance of the plugin for questioning multiple smartmeter
         self.cycle = self.get_parameter_value('cycle')               # the frequency in seconds how often the query shoud be done
         self.host = self.get_parameter_value('snmp_host')            # IP Adress of the network device to be queried
-        self.port = self.get_parameter_value('snmp_port')            # Port for SNMP queries
+        self.port = int(self.get_parameter_value('snmp_port'))       # Port for SNMP queries
```

Comment:
Why do you need to convert to int? Could you possibly use int instead of num in plugin.yaml?